lib/generate-id.js

```diff
@@ -21,12 +20,47 @@ const findId = async ( model, length ) => {
   return id
 }
 
+const idTypes = {
+  shabad: [ Shabads, 3 ],
+  line: [ Lines, 4 ],
+}
+
 /**
- * Generates a line id and a shabad id, using the database.
- */
+  * Logs the id to the console
+  * @param {Number} count the count of id to generate
+  * @param {String} type check `availableTypes`
+  */
+const logGenerateIds = async ( count, type ) => {
+  const newId = idTypes[ type ]
```

Comment:
The name here confused me a little - something like `idType` or `idGenerator` would be better imo